documentation/service-rules/ecf1/lead-provider-api.md

```diff
@@ -2,4 +2,247 @@
 title: Lead provider API
 ---
 
-Lorem ipsum.
+Learn about how our payment engine works once it receives declarations from lead providers using the Early Career Framework API.
+
+These are the steps
+
+1. Provider submits a declaration.
+2. The system validates data.
+3. If invalid, errors are flagged up with corrective action suggestions.
+4. If valid, the declaration is created and linked to an output fee calculation.
+5. Service fees and banding are determined and applied.
+6. Final payments are calculated and stored.
+
+###  What are declarations?
+
+Declarations are submitted where there is evidence of a participant's engagement in training for a given milestone period. This is submitted by lead providers via the API and triggers payments to lead providers.
+
+Declarations serve as the critical "bridge" between proving training engagement and triggering the financial outputs (payments) from the DfE.
+
+Declarations are treated in 'good faith' meaning that providing the declaration is valid and the participant is eligible for funding. They will be paid for.
+
+### How do lead providers submit a declaration?
+
+A lead provider can submit a declaration using the POST `api/v3/participant-declarations` endpoint, providers are required to supply the request body, which includes the:
```

Comment:
`/api/v[1,2,3]/participant-declarations` (or similar) so its clear its in all versions of the API? ditto for the other places we only reference v3